### DISCUSSION TITLE
Calling supercritical CO2 in matlab using coolprop

### DISCUSSION THREAD

**Discussion by fazahmed07** [CoolProp/CoolProp#2331] [General]
I have uploaded a part of matlab code I'm developing for analysing a reversed brayton cycle. I need to call supercritical CO2 from the coolprop library. could someone please help me with this, thanks.
![Screenshot 2023-12-05 003333](https://github.com/CoolProp/CoolProp/assets/65147239/93104845-f77f-40be-8552-57562e45b8dc)

**Comment by ibell** [comment]
In CoolProp there is no distinction between supercritical and non-supercritical CO2. The EOS covers all fluid states.
